lib/operators/param.js

```diff
@@ -1,7 +1,7 @@
 var errors = require("../errors");
 
 module.exports = {
-  parse: function(key, dflt) {
+  parse: function() {
```

Comment:
Can modify the `no-unused-vars` rule http://eslint.org/docs/rules/no-unused-vars to allow unused function params